nautobot/cloud/tests/test_api.py

```diff
@@ -155,63 +155,76 @@ def setUpTestData(cls):
         ]
 
 
+class CloudServiceNetworkAssignmentTest(APIViewTestCases.APIViewTestCase):
+    model = models.CloudServiceNetworkAssignment
+
+    @classmethod
+    def setUpTestData(cls):
+        cloud_networks = models.CloudNetwork.objects.all()[:3]
+        cls.create_data = [
+            {
+                "cloud_network": cloud_networks[0].pk,
+                "cloud_service": models.CloudService.objects.exclude(cloud_networks=cloud_networks[0]).first().pk,
+            },
+            {
+                "cloud_network": cloud_networks[1].pk,
+                "cloud_service": models.CloudService.objects.exclude(cloud_networks=cloud_networks[1]).first().pk,
+            },
+            {
+                "cloud_network": cloud_networks[2].pk,
+                "cloud_service": models.CloudService.objects.exclude(cloud_networks=cloud_networks[2]).first().pk,
+            },
+        ]
+
+
 class CloudServiceTest(APIViewTestCases.APIViewTestCase):
     model = models.CloudService
 
     @classmethod
     def setUpTestData(cls):
         cloud_accounts = models.CloudAccount.objects.all()
-        cloud_networks = models.CloudNetwork.objects.all()
         cloud_resource_types = models.CloudResourceType.objects.get_for_model(models.CloudService).all()
 
         models.CloudService.objects.create(
             name="Deletable Service 1",
             description="It really is deletable",
             cloud_account=cloud_accounts[0],
-            cloud_network=cloud_networks[0],
             cloud_resource_type=cloud_resource_types[0],
         )
         models.CloudService.objects.create(
             name="Deletable Service 2",
-            cloud_network=cloud_networks[1],
             cloud_resource_type=cloud_resource_types[1],
         )
         models.CloudService.objects.create(
             name="Deletable Service 3",
-            cloud_network=cloud_networks[2],
```

Comment:
Was it necessary to remove these but not add some `.cloud_networks.set(...)` in their places?

Reply:
I didn't think it was necessary for the deletable objects since `cloud_networks` isn't a required key anymore. But it's simple enough to add it back

Reply:
That's fair. I guess the factory objects will have cloud_networks values so this is fine.